bin/status-perl

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use 5.010;
+
+use FindBin '$Bin';
+use lib "$Bin/../lib";
+use CPAN2OBS qw/ debug info prompt /;
+
+use Getopt::Long;
+GetOptions(
+    "update" => \my $update,
+    "data=s" => \my $data,
+    "project=s" => \my $project,
+);
+die "--data missing" unless $data;
+$project ||= "home:tinita:CPAN:autoupdate";
+#$project ||= "devel:languages:perl:autoupdate";
+
+my $apiurl = "https://api.opensuse.org";
+
+my $cpan2obs = CPAN2OBS->new({
+    data => $data,
+    apiurl => $apiurl,
+    project_prefix => $project,
+});
+
+my @states = qw/ total done todo new older error /;
+my @status_list;
+
+if ($update) {
+    $cpan2obs->update_status_perl();
+}
+my $states = $cpan2obs->fetch_status_perl();
+my %counts;
+for my $dist (sort keys %$states) {
+    my $dist_status = $states->{ $dist };
+    my $status = $dist_status->[0];
+    if ($status =~ m/^error/) {
+        $status = 'error';
+    }
+    $counts{ $status }++;
+    $counts{total}++;
+    next;
+}
+
+push @status_list, [
+    "perl", map $counts{ $_ } || 0, @states,
```

Comment:
Expression form `map` again. This one being the least readable so far.

Reply:
ok